ellama.el

```diff
@@ -218,8 +231,11 @@ when the request completes (with BUFFER current)."
 		    (goto-char start)
 		    (delete-region start end)
 		    (insert (funcall filter text))
-		    (when (not (derived-mode-p 'prog-mode))
-		      (fill-region start (point)))
+                    (when (pcase ellama-fill-paragraphs
+                            ((cl-type function) (funcall ellama-fill-paragraphs))
+                            ((cl-type boolean) ellama-fill-paragraphs)
+                            ((cl-type list) (derived-mode-p ellama-fill-paragraphs)))
```

Comment:
It doesn't work. `(derived-mode-p '(text-mode))` always return nil.

Reply:
Ah, it looks like support for that calling convention was added in emacs 30. I'll change the calling convention.